src/utils.ts

```diff
@@ -155,7 +155,8 @@ export const execSync = (command: string) =>
 export const getOptionalInput = (name: string) => getInput(name) || undefined
 
 export const getUsername = async (api: Gitlab) => {
```

Comment:
```suggestion
export const getUsername = (api: Gitlab) => {
```

Reply:
If the `async` is removed, the return type of this function becomes `string | Promise<string>`, is this acceptable?

Reply:
I personally accept it, but if the linter complaints, `async` is also fine.

Reply:
The linter is not complaining about it though, maybe I can just remove the `async`.